File: src/main/java/com/github/gitfy/gitfyapi/GitfyApiApplication.java
package com.github.gitfy.gitfyapi;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.EnableAspectJAutoProxy;

@MapperScan(value = "com.github.gitfy.gitfyapi.dao")
@SpringBootApplication
public class GitfyApiApplication {
    public static void main(String[] args) {
        SpringApplication.run(GitfyApiApplication.class, args);
    }
}

File: src/main/kotlin/com/gitfy/gitfyapi/GitfyApiApplication.kt
package com.gitfy.gitfyapi

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
class GitfyApiApplication

fun main(args: Array<String>) {
    runApplication<GitfyApiApplication>(*args)
}
